tests/testapp/models.py

```diff
@@ -114,11 +118,20 @@ class BlogStreamPage(Page):
             ("heading", blocks.CharBlock(classname="full title", icon="title")),
             ("paragraph", blocks.RichTextBlock(icon="pilcrow")),
             ("media", TestMediaBlock(icon="media")),
+            ("video", VideoChooserBlock(icon="media")),
+            ("audio", AudioChooserBlock(icon="media")),
         ]
     )
 
+    featured_media = models.ForeignKey(
+        "wagtailmedia.Media", on_delete=models.PROTECT, related_name="+"
+    )
+
     content_panels = Page.content_panels + [
         FieldPanel("author"),
         FieldPanel("date"),
         StreamFieldPanel("body"),
+        MediaChooserPanel("featured_media"),
+        # MediaChooserPanel("featured_media", media_type="audio"),
```

Comment:
Are these needed? Perhaps we could create separate fields for them?

Reply:
I left them for convenience when checking the interface via `tox -e interactive`
had separate ones but thought that is too much?